src/smk/Window.cpp

```diff
@@ -107,6 +107,11 @@ void GLFWCharCallback(GLFWwindow* glfw_window, unsigned int codepoint) {
 
 }  // namespace
 
+glm::vec2 smk::Window::ToScreenSpace(const glm::vec2& world) {
```

Comment:
The name is a bit confusing, I would have thought we went from screen space to world coordinate.
What about MapPixelToWorldCoordinates?

Reply:
the input is the world position and the output is the screenspace. This is pretty standard convention. Unsure how to make this more clear.

Reply:
MapPixelToWorldCoordinates would be appropriate the more I think about it. SFML uses this name too for that purpose I assume